instrumentation/jedis/jedis-1.4/javaagent/src/main/java/io/lumigo/javaagent/instrumentation/jedis/v1_4/JedisConfigCustomizer.java

```diff
@@ -0,0 +1,45 @@
+/*
```

Comment:
This file was copied from `opentelemetry-java-distro/instrumentation/jdbc/javaagent/src/main/java/io/lumigo/javaagent/instrumentation/jdbc/JdbcConfigCustomizer.java`